### PR TITLE
trailscraper 0.8.0

### PR DESCRIPTION
**Pull request by flosell**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

just released, new version of [trailscraper](https://github.com/flosell/trailscraper), resources generated using ` homebrew-pypi-poet`

### CODE REVIEW

File: Formula/trailscraper.rb [hunk 1]
```diff
@@ -21,31 +20,31 @@ class Trailscraper < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "bd9a48aeb5cd08a08fb10d35d83eebef69c66e9436de70e737e3c10e3efbc3f8"
   end
 
-  depends_on "python@3.11"
```
Comment (chenrui333) [inline Formula/trailscraper.rb:24]: why not `python@3.11`?

**Review by chenrui333 — COMMENTED** [review]
no need to touch `bottle` block, and always remove `revision` for the version bump

**Comment by flosell** [comment]
Got it! I guess my release scripting is no longer up to date...

**Comment by SMillerDev** [comment]
If you're going to script brew pull requests. Please just have it call `brew bump-formula-pr`. That way you won't accidentally overwrite important changes with version bumps.

File: Formula/trailscraper.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/trailscraper.rb:24]: And this is supported in the CI, right?
https://github.com/flosell/trailscraper/commit/510c9abf25c4e81385f4b46618b731417d4c75e4

File: Formula/trailscraper.rb [hunk 2]
```diff
@@ -3,11 +3,10 @@ class Trailscraper < Formula
...
   desc "Tool to get valuable information out of AWS CloudTrail"
   homepage "https://github.com/flosell/trailscraper"
-  url "https://files.pythonhosted.org/packages/f4/89/392581eaa901f2690f5d9b0c9589f41ad03606371f16bedd9680a12413aa/trailscraper-0.7.0.tar.gz"
-  sha256 "8aade831f331d5f3b3780478473c4dbe45dc2018026df0112624cd37bbdc3605"
+  url "https://files.pythonhosted.org/packages/b5/d0/c46ca8b3857576debced3e52471693b5acae996882c9f33a78dc6ddc91da/trailscraper-0.8.0.tar.gz"
+  sha256 "b0ae6be8013226943a6db068420fef8c537c0061cb1e36528b05896aa9326393"
   license "Apache-2.0"
-  revision 1
-  head "https://github.com/flosell/trailscraper.git", branch: "master"
+  head "https://github.com/flosell/trailscraper.git"
```
Comment (SMillerDev) [inline Formula/trailscraper.rb:9]:
```suggestion
  head "https://github.com/flosell/trailscraper.git", branch: "master"
```

Comment (chenrui333) [reply Formula/trailscraper.rb]: 👍

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**